exporter/signalfxexporter/internal/translation/translator.go

```diff
@@ -682,6 +683,65 @@ func aggregateDatapoints(
 		return nil
 	}
 
+	var aggregatedDatapoints []*sfxpb.DataPoint
+
+	// For every data point, need to find every datapoint it matches, so all matches can be aggregated
+	// together. Datapoints can only be used once.
+	indexUsed := make([]bool, len(dps))
+	for i := range indexUsed {
+		indexUsed[i] = false
+	}
```

Comment:
I don't think this is needed, slice items are guaranteed to be initiated with zero values in go

Reply:
Good to know, I'll remove this.